.github/workflows/build-wasm.yml

```diff
@@ -10,27 +10,15 @@ on:
       - 'LICENSE'
       - 'CODEOWNERS'
       - 'renovate.json'
+      - 'query-engine/**'
```

Comment:
Instead of `paths-ignore` we could do the opposite maybe?
```
    paths:
      - 'prisma-fmt-wasm/**'
```

Reply:
The paths would have to include `psl`, `prisma-fmt`, `prisma-fmt-wasm`, parts of `libs` and probably a few top-level files because prisma-fmt-wasm changes based on all of that, so I think we should stick to the exclude-based approach. It's a valid point though, we should think about more aggressively skipping CI when we're sure it is fine to do so.

Reply:
Ok let's keep it "simple" and not change anything for now